src/xrpld/app/tx/detail/VaultClawback.cpp

```diff
@@ -125,7 +254,7 @@ VaultClawback::doApply()
     if (!vault)
         return tefINTERNAL;  // LCOV_EXCL_LINE
 
-    auto const mptIssuanceID = *((*vault)[sfShareMPTID]);
+    auto const mptIssuanceID = vault->getFieldH192(sfShareMPTID);
```

Comment:
Info: You could also write this as
```
auto const mptIssuanceID = *vault->at(sfShareMPTID);
```
Either way is fine. It's mostly just preference.